components/raftstore/src/store/fsm/apply.rs

```diff
@@ -492,6 +496,18 @@ where
             self.kv_wb_last_bytes = 0;
             self.kv_wb_last_keys = 0;
         }
+        if !self.pending_ssts.is_empty() {
+            let tag = self.tag.clone();
+            self.importer
+                .ingest(&self.pending_ssts, &self.engine)
```

Comment:
Importer could only ingest files whose column family are same. So you must group them by cf.

Reply:
Emmm, I see, You group them in method of `impoter`, It is ok.